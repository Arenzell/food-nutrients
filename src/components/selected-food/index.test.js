import React from 'react'
import SelectedFood from './index'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer
    .create(
      <SelectedFood
        selectedFood={{
          food: {
            name: 'Banana',
          },
          amount: 300,
        }}
        onChange={() => {}}
        onRemove={() => {}}
      />,
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
