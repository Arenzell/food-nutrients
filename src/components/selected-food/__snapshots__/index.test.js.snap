// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="selectedFood"
>
  <h2>
    Banana
  </h2>
  <div
    className="selectedFoodAmount"
  >
    <div
      className="ant-input-number"
      style={Object {}}
    >
      <div
        className="ant-input-number-handler-wrap"
      >
        <span
          aria-disabled={false}
          aria-label="Increase Value"
          className="ant-input-number-handler ant-input-number-handler-up "
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchCancel={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="button"
          unselectable="unselectable"
        >
          <i
            aria-label="icon: up"
            className="anticon anticon-up ant-input-number-handler-up-inner"
          >
            <svg
              aria-hidden="true"
              className=""
              data-icon="up"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M890.5 755.3L537.9 269.2c-12.8-17.6-39-17.6-51.7 0L133.5 755.3A8 8 0 0 0 140 768h75c5.1 0 9.9-2.5 12.9-6.6L512 369.8l284.1 391.6c3 4.1 7.8 6.6 12.9 6.6h75c6.5 0 10.3-7.4 6.5-12.7z"
              />
            </svg>
          </i>
        </span>
        <span
          aria-disabled={false}
          aria-label="Decrease Value"
          className="ant-input-number-handler ant-input-number-handler-down "
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchCancel={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="button"
          unselectable="unselectable"
        >
          <i
            aria-label="icon: down"
            className="anticon anticon-down ant-input-number-handler-down-inner"
          >
            <svg
              aria-hidden="true"
              className=""
              data-icon="down"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
              />
            </svg>
          </i>
        </span>
      </div>
      <div
        aria-valuemax={5000}
        aria-valuemin={0}
        aria-valuenow={300}
        className="ant-input-number-input-wrap"
        role="spinbutton"
      >
        <input
          autoComplete="off"
          className="ant-input-number-input"
          max={5000}
          min={0}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseUp={[Function]}
          required={false}
          step={1}
          type="number"
          value="300"
        />
      </div>
    </div>
    <button
      className="ant-btn deleteBtn"
      onClick={[Function]}
      type="button"
    >
      <i
        aria-label="icon: delete"
        className="anticon anticon-delete"
      >
        <svg
          aria-hidden="true"
          className=""
          data-icon="delete"
          fill="currentColor"
          focusable="false"
          height="1em"
          viewBox="64 64 896 896"
          width="1em"
        >
          <path
            d="M360 184h-8c4.4 0 8-3.6 8-8v8h304v-8c0 4.4 3.6 8 8 8h-8v72h72v-80c0-35.3-28.7-64-64-64H352c-35.3 0-64 28.7-64 64v80h72v-72zm504 72H160c-17.7 0-32 14.3-32 32v32c0 4.4 3.6 8 8 8h60.4l24.7 523c1.6 34.1 29.8 61 63.9 61h454c34.2 0 62.3-26.8 63.9-61l24.7-523H888c4.4 0 8-3.6 8-8v-32c0-17.7-14.3-32-32-32zM731.3 840H292.7l-24.2-512h487l-24.2 512z"
          />
        </svg>
      </i>
    </button>
  </div>
</div>
`;
